src/Cardano/Wallet/Mnemonic/GenerateMnemonic.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE DataKinds #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- | Mnemonic generation executable
+
+
+import Prelude
+
+import Cardano.Wallet.Mnemonic
+    ( Mnemonic, entropyToMnemonic, genEntropy )
+import Data.Aeson
+    ( encode )
+
+main :: IO ()
+main = do
+    backupPhrase <- generateBackupPhrase
+    let backupPhraseString = backupPhraseToString backupPhrase
+    putStrLn backupPhraseString
```

Comment:
Minor note: would be better to actually output a `Text` or a `ByteString` here. In practice, we probably _never_ wants to output strings. They are inefficient (list of chars) and don't play well in multi-thread contexts.

Reply:
used `Text`